### ISSUE TITLE
Error when installing from wiki instructions

### ISSUE THREAD

**Issue by robertritz** [mysociety/yournextrepresentative#948]
I ran into an issue when installing ynr from the [production deployment [wiki](https://github.com/mysociety/yournextrepresentative/wiki/Production-Deploy).] I got around the issue but wanted to let y'all know.

When running `pip install -r requirements.txt` Pillow failed to install. I used easy_install and Pillow install with no issues. I reran `pip install -r requirements.txt` and everything installed properly.

**Comment by robertritz** [comment]
I tried to replicate the solution later but couldn't. It appears the real issue is that I was missing libjpeg. I installed libjpeg-dev and it worked. Can anyone verify this is a needed resource?

**Comment by wfdd** [comment]
Yes, libjpeg-dev is required.  For an up-to-date list of system dependencies, bar PostgreSQL and elasticsearch, see https://github.com/mysociety/yournextrepresentative/blob/master/conf/provision.sh#L7-L11.
